OlsztynHooligans/angular.json

```diff
@@ -23,7 +23,7 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "src/tsconfig.app.json",
             "assets": [
-              "src/favicon.ico",
+              "src/favA.ico",
```

Comment:
dlaczego favA?

Reply:
poprawione to co było źle, poproszę o zatwierdzenie